TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -92,24 +92,27 @@ private IEnumerable<CodeElementWrapper> CodeElementFinder(FileCompiler fileCompi
                 while (codeElements.Count == 0 && lineIndex >= 0)
                 {
                     var codeElementsLine =
-                        fileCompiler.CompilationResultsForProgram.ProgramClassDocumentSnapshot.PreviousStepSnapshot.Lines[lineIndex];
+                        fileCompiler.CompilationResultsForProgram.ProgramClassDocumentSnapshot.PreviousStepSnapshot
```

Comment:
No changes in this file, you can revert to the original source.

Reply:
That fact is I didn't touch this source.

Reply:
I think @LANOYDO wanted to change something at first and then came back to the original code but kept indent/style changes.